tools/swc-transform-css-modules/package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "@vkontakte/vkui-swc-transform-css-modules",
+  "version": "0.1.0",
+  "description": "",
+  "author": {
+    "name": "Daniil Suvorov",
+    "email": "severecloud@gmail.com",
+    "url": "https://github.com/severecloud"
+  },
+  "license": "MIT",
+  "keywords": [
+    "swc-plugin"
+  ],
+  "main": "swc_transform_css_modules.wasm",
+  "scripts": {
+    "prepack": "yarn run build-wasi",
+    "build-wasi": "cargo build-wasi --release",
+    "postbuild-wasi": "cp target/wasm32-wasi/release/swc_transform_css_modules.wasm .",
+    "build": "echo use cargo build-wasi",
+    "test": "echo use cargo test",
```

```suggestion
    "test": "cargo test",
```
из-за времени холодного запуска не будем так объявлять?

---

С большой вероятностью при запуске всех тесто в монорепе, разработчики получат ошибку `command not found: cargo`. Кажется не стоит заставлять всех устанавливать rust

---

Я бы тогда в отдельной репе хранил специфичные плагины

---

Решили хранить в репе